tdrs-backend/tdpservice/data_files/admin/admin.py

```diff
@@ -23,11 +26,47 @@ def has_change_permission(self, request, obj=None):
         """Read only permissions."""
         return False
 
-
 @admin.register(DataFile)
 class DataFileAdmin(ReadOnlyAdminMixin, admin.ModelAdmin):
     """Admin class for DataFile models."""
 
+    class Media:
+        """Media class for DataFileAdmin."""
+
+        js = ('admin/js/admin/admin_datafile_model.js',)
+
+    actions = ['reparse']
+
+    def reparse(self, request, queryset):
```

Comment:
we should be able to remove the `reparse` command since jquery intersects the command run and then calls view function

Reply:
I just tested the latest and the view function is not what is being called. This action is what is called.